apps/studio/components/interfaces/LogDrains/LogDrainDestinationSheetForm.tsx

```diff
@@ -446,13 +460,41 @@ export function LogDrainDestinationSheetForm({
           )}
         </SheetSection>
 
-        <SheetSection></SheetSection>
+        <div className="mt-auto">
+          <SheetSection className="border-t bg-background-alternative-200 mt-auto">
+            <FormItemLayout
+              isReactForm={false}
+              layout="horizontal"
+              label={
+                <div className="text-foreground-light">
+                  Additional drain cost
+                  <div className="text-foreground-lighter mt-2">
+                    <Link
+                      target="_blank"
+                      rel="noopener noreferrer"
+                      className="underline flex gap-1 items-center"
+                      href="https://supabase.com/docs/guides/platform/log-drains"
+                    >
+                      Documentation <ExternalLink className="w-4 h-4" />
+                    </Link>
+                  </div>
+                </div>
+              }
+            >
+              <ul className="text-right text-foreground-light">
+                <li className="text-brand-link text-base">$60 / drain / month</li>
```

Comment:
```suggestion
                <li className="text-brand-link text-base">$60 per drain per month</li>
```

Reply:
@kevcodez any preference? went with what is in the pricing page

Reply:
No hard feelings - given this is inside of a modal, can also use the shorter form with a slash